website/src/lib/ipfs.ts

```diff
@@ -1,7 +1,7 @@
 import { create } from 'kubo-rpc-client';
 
 const ipfsClient = create({
-  url: 'https://api.thegraph.com/ipfs/api/v0',
+  url: 'https://ipfs.thegraph.com/ipfs/api/v0',
```

Comment:
There's no `/ipfs/api/v0` (we only allow that atm. for backwards compatibility reasons). This should just be `https://ipfs.thegraph.com/api/v0` if the client expects the full Kubo RPC path.